### PR TITLE
LPS-144911 Set fieldName column's type from VARCHAR(255) to TEXT in a new upgrade process

### PR DESCRIPTION
**Pull request by IstvanD**
Related Issue: https://issues.liferay.com/browse/LPS-144911

Dear Team,

I modified the solution so it addresses Brian's concerns in https://github.com/brianchandotcom/liferay-portal/pull/112555#issuecomment-1020521780

`FieldNameUpgradeProcess` sets the `fieldName` column's type explicitly to TEXT if it not already has that type.
 
Thank you and best regards,
István

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @IstvanD, can you please check if these failures are related to your changes? Thanks

**Comment by IstvanD** [comment]
Dear Team,

The test failures are not related to the solution.

Best regards,
István

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_1_3/util/DDMFieldTable.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import java.sql.Types;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author	  Brian Wing Shun Chan
+ * @generated
+ */
+public class DDMFieldTable {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_1_3/util/DDMFieldTable.java:26]: I didn't know this task, @IstvanD! Great job! :clap: :tada:

**Comment by natocesarrego** [comment]
ci:forward

**Comment by liferay-forms** [comment]
Hey @IstvanD, can you please check if these failures are related to your changes? Thanks

**Comment by IstvanD** [comment]
Dear Team,

The test failures are not related to the solution.

Best regards,
István

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by liferay-forms** [comment]
Hey @IstvanD, can you please check if these failures are related to your changes? Thanks

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_1_3/util/DDMFieldTable.java (hunk 1 above)
Comment (IstvanD) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_1_3/util/DDMFieldTable.java:26]: Thank you @natocesarrego! I also got the idea from someone else :)

**Comment by IstvanD** [comment]
Dear Team,

The test failures are not related to the solution.

Best regards,
István

**Comment by natocesarrego** [comment]
ci:reevaluate:1257401_1470

**Comment by liferay-forms** [comment]
Hey @IstvanD, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**